data/bingo/functions/init/init.mcfunction

```diff
@@ -48,13 +48,13 @@ forceload add 0 0
 	# 	inventory. Should look like this: "clear @a[tag=bingo.clear] <item> 1
 	# categories: (String List) List of category names. All categories referenced
 	# 	here should be registered separately.
-	# weight: (Integer) Weight of this item for the rng.
+	# weight: (Integer) Weight of this item for the RNG.
```

Comment:
This one is especially philosophical and in doc comment doesn't matter. No need to undo it, just also no need to think about what the "more correct" capitalization might be.